<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilesUtil</class>
 <widget class="QWidget" name="FilesUtil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>742</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>下载</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>190</y>
     <width>492</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_downloadFile" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>661</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_DownloadFiles">
    <item>
     <widget class="QLabel" name="label_fileName">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>文件名</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar_Download">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_cancelDownload">
      <property name="text">
       <string>取消下载</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_pauseDownload">
      <property name="text">
       <string>暂停下载</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkBox_downloadFinished">
      <property name="text">
       <string>下载完成</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_openFromLocalFile">
      <property name="text">
       <string>在文件夹中显示</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
